spec/item_spec.rb

```diff
@@ -0,0 +1,45 @@
+require './item'
+require './label'
+require './genre'
+require './author'
+
+describe Item do
+  item1 = Item.new('20/10/2020')
+
+  describe 'if a parameter passed correctly' do
+    it 'should return the correct parameter' do
+      expect(item1.publish_date).to eq '20/10/2020'
+    end
+  end
+
+  describe '#add_label' do
+    it 'should add the label correctly' do
+      label1 = Label.new('Education', 'Blue')
+      item1.add_label(label1)
+      expect(item1.label).to be label1
+    end
+  end
+
+  describe '#add_author' do
+    it 'should add the label correctly' do
+      author1 = Author.new('Ellen', 'White')
+      item1.add_author(author1)
+      expect(item1.author).to be author1
+    end
+  end
+
+  describe '#add_genre' do
+    it 'should add the label correctly' do
+      genre1 = Genre.new('Spiritual')
+      item1.add_genre(genre1)
+      expect(item1.genre).to be genre1
+    end
+  end
+
+  describe '#archive if older than 10 years' do
+    it 'should return true' do
+      item1.publish_date = '20/10/2010'
+      expect(item1.can_be_archived?).to be true
+    end
+  end
+end
```

Comment:
- [ ] Great work so far but I guess there is some misunderstanding 😨 Because the previous reviewer asked you to create a test for the `move_to_archive` method 🤔 You have a test for the `can_be_archived?` method which is good but kindly make a test case for `move_to_archive` to test your project well as suggested by the previous reviewer 👍

Reply:
Hi @Hammas, yes sorry my mistake. Will add the move_to_archive test.